db/migrate/20180216100846_add_subscription_contents_index.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubscriptionContentsIndex < ActiveRecord::Migration[5.1]
+  def change
+    add_index :subscription_contents, :digest_run_subscriber_id
```

Comment:
Do you know how long this takes and if we have to add the index concurrently?

Reply:
No... But I do have a DB backup now so I'll find out.

Reply:
It was 0.1218s on my machine with 262751 `SubscriptionContent` records.

Reply:
That seems fine to me.